cypress/e2e/Courses/Certificates/StandardPlanUnawardedCourseCertificate.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Unawarded Premium Course Certificates for Standard Plan Users
+  Background:
+    Given a standard plan user has some completed unawarded premium courses
+    And the premium courses page has been navigated to
+
+  Scenario: Standard plan users should see the call to action "Review course" for completed and unawarded premium courses
+    Given an iOS mobile device is not being used
+    When the user selects the "Completed" tab
+    Then the user should see for unawarded courses cards the warning style
+    And should see the call to action "Review course"
+
+  Scenario: Standard plan users can successfully go to "Review course"
+    Given an iOS mobile device is not used
+    And the "Completed" tab is selected
+    And the user should see for unawarded courses cards the warning style
+    And the call to action "Review course" is displayed to the user
+    When the user click to "Review course"
+    Then the user should be taken to "Review course"
+
+  Scenario: Standard plan users with unawarded course credits can not download certificate
```

Comment:
```suggestion
  Scenario: Standard plan users should see the option to contact support for unawarded premium courses
```

Please use the last version of the gherkin, assuming the last implementation is enabled (FF on).

Reply:
![image](https://github.com/bixlabs/CypressAcapediaProfessionals/assets/6330366/84366407-b9d7-4e0f-a343-8a94101c4b86)